WordPressKit/BlogServiceRemoteREST.m

```diff
@@ -95,6 +95,62 @@ - (void)getAuthorsWithSuccess:(UsersHandler)success
                           }];
 }
 
+- (void)getAllAuthorsWithNumber:(NSNumber *)number
+                        success:(UsersHandler)success
+                        failure:(void (^)(NSError *error))failure
+{
+    NSMutableArray *remoteUsers = [NSMutableArray array];
+    [self getAllAuthorsWithRemoteUsers:remoteUsers
+                                number:number
+                                offset:nil
+                               success:success
+                               failure:failure];
+}
+
+- (void)getAllAuthorsWithRemoteUsers:(NSMutableArray <RemoteUser *>*)remoteusers
```

Comment:
Same comment about "all authors".  Maybe just 'getAuthors' since the number can be limited.